# Generated for NaoQi method ALFrameManager.playDefaultProject
# Play default behavior

---


